### PR TITLE
Extend trajectory export tool to write TransformStamped topics.

### PR DESCRIPTION
**Pull request by MichaelGrupp**
See discussion in #1166.

### CODE REVIEW

**Comment by MichaelGrupp** [comment]
Side effect: with this you can now load the bag directly into evo tools since yesterday ;)

File: cartographer_ros/cartographer_ros/dev/pbstream_trajectories_to_rosbag_main.cc [hunk 1]
```diff
@@ -40,15 +40,15 @@ geometry_msgs::TransformStamped ToTransformStamped(
     const std::string& child_frame_id,
     const cartographer::transform::proto::Rigid3d& parent_T_child) {
   static int64_t seq = 0;
-  geometry_msgs::TransformStamped xfm;
-  xfm.header.seq = ++seq;
-  xfm.header.frame_id = parent_frame_id;
-  xfm.header.stamp = cartographer_ros::ToRos(
+  geometry_msgs::TransformStamped transform_stamped;
```
Comment (MichaelGrupp) [inline cartographer_ros/cartographer_ros/dev/pbstream_trajectories_to_rosbag_main.cc]: changed the name here, let me know if you like xfm better

File: cartographer_ros/cartographer_ros/dev/pbstream_trajectories_to_rosbag_main.cc [hunk 2]
```diff
@@ -40,15 +40,15 @@ geometry_msgs::TransformStamped ToTransformStamped(
     const std::string& child_frame_id,
     const cartographer::transform::proto::Rigid3d& parent_T_child) {
   static int64_t seq = 0;
-  geometry_msgs::TransformStamped xfm;
-  xfm.header.seq = ++seq;
```
[the remark this answers is not in the document]
Comment (MichaelGrupp) [reply cartographer_ros/cartographer_ros/dev/pbstream_trajectories_to_rosbag_main.cc]: Yes. @sebastianklose do you need `seq` or can I remove it?

[the remark this answers is not in the document]
Comment (sebastianklose) [reply cartographer_ros/cartographer_ros/dev/pbstream_trajectories_to_rosbag_main.cc]: Yes a publisher would overwrite, but AFAIK the rosbag lib write(...) won't. Please leave it in for now.

[the remark this answers is not in the document]
Comment (MichaelGrupp) [reply cartographer_ros/cartographer_ros/dev/pbstream_trajectories_to_rosbag_main.cc]: fine for me

**Comment by sebastianklose** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user sebastianklose. Merge queue now has a length of 1.

**Pull request merged**